en/docs/reference/connectors/mongodb-connector/3.x/mongodb-connector-config.md

```diff
@@ -0,0 +1,1981 @@
+            </td>
+            <td>
+                true
+            </td>
+            <td>
+                No
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Heartbeat Frequency MS
+            </td>
+            <td>
+                Number
+            </td>
+            <td>
+                Controls the intervals between which the driver checks the state of the MongoDB deployment. The interval (in milliseconds) between checks, counted from the end of the previous check until the beginning of the next one.
+            </td>
+            <td>
+                -
+            </td>
+            <td>
+                No
+            </td>
+        </tr>
+        <tr>
+            <td>
+                App Name
+            </td>
+            <td>
+                String
+            </td>
+            <td>
+                Specify a custom app name.
+            </td>
+            <td>
+                -
+            </td>
+            <td>
+                No
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Retry Reads
+            </td>
+            <td>
+                Boolean
+            </td>
+            <td>
+                Enables retryable reads.
+            </td>
+            <td>
+                -
+            </td>
+            <td>
+                No
+            </td>
+        </tr>
+        <tr>
+            <td>
+                Retry Writes
+            </td>
+            <td>
+                Boolean
+            </td>
+            <td>
+                Enable retryable writes.
+            </td>
+            <td>
+                -
+            </td>
+            <td>
+                No
+            </td>
+        </tr>
+        <tr>
+            <td>
+                UUID Representation
+            </td>
+            <td>
+                String
+            </td>
+            <td>
+                The type of UUID representation.
+            </td>
+            <td>
+                -
+            </td>
+            <td>
+                No
+            </td>
+        </tr>
+        
+    </table>
+
+**Sample Configuration of Connection URI**
+
+    ```xml
+    <?xml version="1.0" encoding="UTF-8"?>
+    <localEntry key="uriConnection" xmlns="http://ws.apache.org/ns/synapse">
+        <mongodb.init>
+            <name>uriConnection</name>
+            <connectionType>MONGODB</connectionType>
+            <inputType>Connection String URI</connectionType>
+            <connectionURI>mongodb+srv://server.example.com/?connectTimeoutMS=300000&authSource=aDifferentAuthDB</connectionURI>
+            <database>users</database>
+        </mongodb.init>
+    </localEntry>
+    ```
```

Comment:
This will not render properly. We may need to remove the tab indentation here. Check other places as well.

```suggestion
```xml
<?xml version="1.0" encoding="UTF-8"?>
<localEntry key="uriConnection" xmlns="http://ws.apache.org/ns/synapse">
    <mongodb.init>
        <name>uriConnection</name>
        <connectionType>MONGODB</connectionType>
        <inputType>Connection String URI</connectionType>
        <connectionURI>mongodb+srv://server.example.com/?connectTimeoutMS=300000&authSource=aDifferentAuthDB</connectionURI>
        <database>users</database>
    </mongodb.init>
</localEntry>
```
```

Reply:
Fixed